db/migrate/20231222072450_add_info_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddInfoToUsers < ActiveRecord::Migration[7.1]
+  def change
+    add_column :users, :uid, :string
```

Comment:
When migrating 2 or more fields of a table, you should use:
```
change_table(:users, bulk: true) do |t|
end
```

Reply:
em đã sửa